src/apps/api/serializers.py

```diff
@@ -111,6 +115,33 @@ def get_pub_excerpts(self, obj):
 
         return serializer.data
 
+    def get_suggestions_count(self, obj):
+        group = obj.invited_groups.filter(public_participation=True).first()
+        return group.suggestions.count()
+
+    def get_vote_count(self, obj):
+        group = obj.invited_groups.filter(public_participation=True).first()
+        suggestions = group.suggestions.all()
+        total_votes = 0
+        for suggestion in suggestions:
```

Comment:
Usa o Annotate em suggestion ao invés de usar o "for"

Reply:
Ok, já corrigi isso,